src/transformations/AddCommentsTransformation.cpp

```diff
@@ -0,0 +1,79 @@
+#include "../../include/transformations/AddCommentsTransformation.h"
+
+using clang::IfStmt, clang::ForStmt, clang::WhileStmt;
+using llvm::isa, llvm::cast;
+using std::unique_ptr, std::find, std::vector, std::string;
+
+// ------------ AddCommentsVisitor ------------
+
+AddCommentsVisitor::AddCommentsVisitor(Rewriter * rewriter, vector<string> statements) :
+                                       rewriter(rewriter), statements(move(statements)) {}
+
+bool AddCommentsVisitor::VisitStmt(Stmt *s) {
+    if (isa<IfStmt>(s)) {
+        auto IfStatement = cast<IfStmt>(s);
+        if (containStatement(ifCondition)) {
+            rewriter->InsertText(IfStatement->getCond()->getExprLoc(), "/* 'if' cond */",
+                                 true, false);
```

Comment:
What are these arguments doing? I'm confused because you use both `true` in other `InsertText` calling.